pkg/mirror/service.go

```diff
@@ -161,7 +161,7 @@ func (p *pullThroughMirror) RetrieveProviderArchive(ctx context.Context, provide
 }
 
 func (p *pullThroughMirror) upstreamSha256Sums(ctx context.Context, provider *core.Provider, versions *core.ProviderVersions) (*core.Sha256Sums, error) {
-	if len(versions.Versions) == 0 || len(versions.Versions[0].Platforms) == 0 {
+	if len(versions.Versions) == 0 {
 		return nil, errors.New("core.ProviderVersions doesn't contain any platforms")
```

Comment:
Can we change the error message to `"core.ProviderVersions doesn't contain any versions"` please?

Reply:
Done.